xla/service/while_loop_unroller.cc

```diff
@@ -226,42 +221,39 @@ UnrollSingleIterationOfTrivialLoop(HloInstruction* while_op,
 // 2. trip count.
 // 3. unroll expansion limit (#_body_instructions * trip_count).
 // These conditions can be changed per usecase.
-bool InitialFeasibilityCheck(HloInstruction* while_op, WhileLoopConfig config) {
+bool InitialFeasibilityCheck(HloInstruction* while_op, WhileLoopConfig config, UnrollConfig unroll_config) {
```

Comment:
nit: seems like this could be a const ref?

Reply:
@ptoulme-aws please fix this